django-entrypoint.sh

```diff
@@ -14,10 +14,11 @@ if [ "$1" = 'celery' ]; then
 fi
 
 if [ "$1" = 'gunicorn' ]; then
-  # Do an extra chown of the /app directory at runtime in addition to the one in
-  # the build process in case any directories are mounted as root-owned volumes
```

Comment:
There hasn't been a chown during the build process for a while now. That was when we had an onbuild image.